pkg/tfgen/installation_docs.go

```diff
@@ -106,24 +110,50 @@ func writeFrontMatter(providerDisplayName string) string {
 // .NET: Pulumi.foo
 // Java: com.pulumi/foo
 // ****
-func writeInstallationInstructions(goImportBasePath, displayName, pkgName string) string {
+//
+// A dynamically bridged provider receives the following instead:
+// ****
+// ## Generate Provider
+//
+// The Foo provider must be installed as a Local Package by following the instructions for Any Terraform Provider: (link)
+// ```bash
+// pulumi package add terraform-provider org/foo
+// ```
+// ****
```

Comment:
Go's preferred way to add examples is with indentation.
```suggestion
//	## Generate Provider
//	
//	The Foo provider must be installed as a Local Package by following the instructions for Any Terraform Provider: (link)
//	 ```bash
//	pulumi package add terraform-provider org/foo
//	```
```